Buildings/Fluid/HeatExchangers/ThermalWheels/Latent/BaseClasses/PartialWheel.mo

```diff
@@ -88,67 +48,44 @@ partial model PartialWheel
 
 protected
   Buildings.Fluid.HeatExchangers.ThermalWheels.Latent.BaseClasses.Effectiveness effCal(
-    final epsSenCoo_nominal=epsSenCoo_nominal,
-    final epsLatCoo_nominal=epsLatCoo_nominal,
-    final epsSenCooPL=epsSenCooPL,
-    final epsLatCooPL=epsLatCooPL,
-    final epsSenHea_nominal=epsSenHea_nominal,
-    final epsLatHea_nominal=epsLatHea_nominal,
-    final epsSenHeaPL=epsSenHeaPL,
-    final epsLatHeaPL=epsLatHeaPL,
-    final mSup_flow_nominal=mSup_flow_nominal)
+    final epsSen_nominal=per.epsSen_nominal,
+    final epsLat_nominal=per.epsLat_nominal,
+    final epsSenPL=per.epsSenPL,
+    final epsLatPL=per.epsLatPL,
+    final mSup_flow_nominal=per.mSup_flow_nominal)
     "Calculate the effectiveness of heat exchanger"
     annotation (Placement(transformation(extent={{-100,-10},{-80,10}})));
   Buildings.Fluid.HeatExchangers.ThermalWheels.Latent.BaseClasses.HeatExchangerWithInputEffectiveness hex(
     redeclare package Medium1 = Medium,
     redeclare package Medium2 = Medium,
-    final m1_flow_nominal=mSup_flow_nominal,
-    final m2_flow_nominal=mExh_flow_nominal,
-    final dp1_nominal=dpSup_nominal,
-    final dp2_nominal=dpExh_nominal)
+    final m1_flow_nominal=per.mSup_flow_nominal,
+    final m2_flow_nominal=per.mExh_flow_nominal,
+    final dp1_nominal=per.dpSup_nominal,
+    final dp2_nominal=per.dpExh_nominal)
     "Heat exchanger"
     annotation (Placement(transformation(extent={{10,-10},{30,10}})));
-  Modelica.Blocks.Sources.RealExpression TSup(
-    final y(final unit="K")=Medium.temperature(
-      Medium.setState_phX(
-        p=port_a1.p,
-        h=inStream(port_a1.h_outflow),
-        X=inStream(port_a1.Xi_outflow))))
-    "Supply air temperature"
-    annotation (Placement(transformation(extent={{-160,18},{-140,38}})));
-  Modelica.Blocks.Sources.RealExpression TExh(
-    final y(final unit="K")=Medium.temperature(
-      Medium.setState_phX(
-        p=port_a2.p,
-        h=inStream(port_a2.h_outflow),
-        X=inStream(port_a2.Xi_outflow))))
-    "Exhaust air temperature"
-    annotation (Placement(transformation(extent={{-160,-40},{-140,-20}})));
+
+initial equation
```

Comment:
There is no need to have the `assert()` function, if we can finalize the flag `haveLatentHeatExchange`.